hls4ml/model/types.py

```diff
@@ -300,7 +300,7 @@ def saturation_mode(self, mode):
     def __str__(self):
         args = [self.width, self.integer, self.rounding_mode, self.saturation_mode, self.saturation_bits]
         args = ','.join([str(arg) for arg in args if arg is not None])
-        typestring = '{signed}fixed<{args}>'.format(signed='u' if not self.signed else '', args=args)
+        typestring = 'ap_{signed}fixed<{args}>'.format(signed='u' if not self.signed else '', args=args)
```

Comment:
We don't have 'ap_' or 'ac_' prefixes in our internal typestings. This change should be reverted

Reply:
I have removed the `ap_` here and fixed the vivado_accelerator writer which gave issues with the compiler due to the `my_project_axi.hpp` not using the hls type definition but directly the internal typestrings. However this fix has been removed from this branch to separate the fixes.